crates/oxc_linter/src/rules/jsx_a11y/aria_proptypes.rs

```diff
@@ -497,15 +504,13 @@ fn test() {
         r#"<div aria-hidden="yes" />"#,
         r#"<div aria-hidden="no" />"#,
         "<div aria-hidden={1234} />",
-        "<div aria-hidden={`${abc}`} />",
         "<div aria-label={true} />",
         "<div aria-label={false} />",
         "<div aria-label={1234} />",
         "<div aria-label={!true} />",
         r#"<div aria-checked="yes" />"#,
         r#"<div aria-checked="no" />"#,
         "<div aria-checked={1234} />",
-        "<div aria-checked={`${abc}`} />",
```

Comment:
The OG set this rules as invalid:
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/__tests__/src/rules/aria-proptypes-test.js#L242